content/actions/publishing-packages/publishing-nodejs-packages.md

```diff
@@ -180,3 +180,9 @@ jobs:
         env:
           NODE_AUTH_TOKEN: {% raw %}${{ secrets.NPM_TOKEN }}{% endraw %}
 ```
+
+To ensure authentication with the registry during publishing add a `npmAuthToken` entry to the `.yarnrc.yml`.
+
+```yaml
+npmAuthToken: '${NODE_AUTH_TOKEN:-}'
+```
```

Comment:
```suggestion

To authenticate with the registry during publishing, ensure your authentication token is also defined in your `yarnrc.yml` file. For more information, see the [Settings](https://yarnpkg.com/configuration/yarnrc#npmAuthToken) article in the Yarn documentation.
```
I think this is very useful information to call out! However, I am hesitant to document the exact Yarn syntax in the GitHub docs. If Yarn updates the syntax they use in their `.yarnrc` file (like they did when they replaced `_authToken` with `npmAuthToken`), this doc will contain incorrect information and be out of date. I'd rather link to the Yarn docs so people can get the authentication instructions directly from them.